ggshield/core/config/auth_config.py

```diff
@@ -137,3 +152,6 @@ def get_instance_token(self, instance_name: str) -> str:
         if instance.account is None:
             raise MissingTokenError(instance=instance_name)
         return instance.account.token
+
+
+AuthConfigSchema = marshmallow_dataclass.class_schema(AuthConfig)
```

Comment:
I think we could use the decorators `@dataclass` https://pypi.org/project/marshmallow-dataclass/ to simplify the code

Reply:
Ha, we already have a dataclass decorator, not sure how those two would interact with each other